File: 프로그래머스/2/42584. 주식가격/주식가격.js
function solution(prices) {
    const answer = [];
    
    for(let i = 0; i < prices.length; i++){
        let time = 0;
        
        // 마지막 index인 경우, 0초 return
        if(i === prices.length-1){
            answer.push(time);
            break;
        }
        
        // 현재 index 이후의 값들과 비교
        for(let j = i+1; j < prices.length; j++){
            if(prices[j] >= prices[i]){
                time++;
                continue;
            }else{
                time++;
                break;
            }
        }
        answer.push(time);
    }
    return answer;
}

File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = []
    
    for idx, price in enumerate(prices):
        time = 0
        
        for i in range(idx+1, len(prices)):
            time += 1
            
            if(price <= prices[i]):
                continue
            else:
                break
                
        answer.append(time)
        
    return answer